src/main/java/com/example/demo1/HelloApplication.java

```diff
@@ -0,0 +1,298 @@
+        Rotate xRotate;
+        Rotate yRotate;
+        group.getTransforms().addAll(
+                xRotate = new Rotate(0, Rotate.X_AXIS),
+                yRotate = new Rotate(0, Rotate.Y_AXIS)
+        );
+        xRotate.angleProperty().bind(angleX);
+        yRotate.angleProperty().bind(angleY);
+
+        scene.setOnMousePressed(event -> {
+            anchorX = event.getSceneX();
+            anchorY = event.getSceneY();
+            anchorAngleX = angleX.get();
+            anchorAngleY = angleY.get();
+        });
+
+        scene.setOnMouseDragged(event -> {
+            angleX.set(anchorAngleX - (anchorY - event.getSceneY()));
+            angleY.set(anchorAngleY + anchorX - event.getSceneX());
+        });
+
+        stage.addEventHandler(ScrollEvent.SCROLL, event -> {
+            double delta = event.getDeltaY();
+            group.translateZProperty().set(group.getTranslateZ() + delta);
+        });
+    }
+
+    private void loadVerticesFromFile() {
+        FileChooser fileChooser = new FileChooser();
+        fileChooser.setTitle("Open Vertices File");
+        File file = fileChooser.showOpenDialog(null);
+
+        if (file != null) {
+            try (Scanner scanner = new Scanner(file)) {
+                List<Float> vertexList = new ArrayList<>();
+                while (scanner.hasNextLine()) {
+                    String line = scanner.nextLine().trim();
+                    String[] values = line.split("\\s+");
+
+                    for (int i = 1; i < values.length; i++) {
+                        try {
+                            vertexList.add(Float.parseFloat(values[i]));
+                        } catch (NumberFormatException e) {
+                            System.out.println("Error parsing number: " + values[i]);
+                        }
+                    }
+                }
+                vertices = new float[vertexList.size()];
+                for (int i = 0; i < vertexList.size(); i++) {
+                    vertices[i] = vertexList.get(i);
+                }
+                System.out.println("Vertices loaded successfully.");
+                System.out.println("Vertices length is " + vertices.length);
+
+                // Для отладки - выводим массив вершин
+                for (float i : vertices) {
+                    System.out.println(i);
+                }
+            } catch (FileNotFoundException e) {
+                e.printStackTrace();
+            }
+        }
+
+        texture = new float[vertices.length / 3 * 2];
+        for (int i = 0; i < texture.length; i += 2) {
+            texture[i] = 0;    // Значение u
+            texture[i + 1] = 0; // Значение v
+        }
+        System.out.println("Textures array");
+        System.out.println("Texture length is " + vertices.length);
+        for (float i : texture) {
+            System.out.println(i);
+        }
+    }
+
+    private void loadFacesFromFile() {
```

Comment:
Propose to move all methods related to reading from files to a separate class. We can call it FileReader or something like that